scripts/migration/migrate_registered_meta.py

```diff
@@ -0,0 +1,116 @@
+"""
+Changes existing registered_meta on a node to new schema layout
+required for the prereg-prize
+"""
+from nose.tools import *  # noqa PEP8 asserts
+import json
+import sys
+import logging
+
+from modularodm import Q
+from modularodm.exceptions import NoResultsFound
+
+from framework.mongo.utils import from_mongo
+from framework.transactions.context import TokuTransaction
+
+from website.models import Node, MetaSchema
+from website.app import init_app
+from website.project.model import ensure_schemas
+from website.project.metadata.schemas import _name_to_id
+
+from scripts import utils as scripts_utils
+
+logger = logging.getLogger(__name__)
+logger.setLevel(logging.INFO)
+
+def verify_migration(nodes, dev):
+    for node in nodes:
+        try:
+            assert_is_not_none(node.registered_meta)
+            assert_is_not_none(node.registered_schema)
+            schema = node.registered_schema
+            assert_greater(schema.version, 1)
+        except AssertionError as e:
+            if dev:
+                logger.info("Node: {0} has no associated registered_schema.".format(node._id))
+            else:
+                raise e
+
+def get_old_registered_nodes():
+    # nullify old registered_schema refs
+    MetaSchema.remove(Q('schema_version', 'eq', 1))
+    ensure_schemas()
+
+    return Node.find(
+        Q('is_registration', 'eq', True)
+    )
+
+def main(dry_run, dev=False):
+    init_app(routes=False)
+    count = 0
+    skipped = 0
+    if not dry_run:
+        scripts_utils.add_file_logger(logger, __file__)
+        logger.info("Iterating over all registrations")
+
+    nodes = get_old_registered_nodes()
+    for node in nodes:
+        schemas = node.registered_meta
+        if not schemas:
+            logger.info('Node: {0} is registered but has no registered_meta'.format(node._id))
+            schemas = {}
+            node.registered_meta = {}
+        # there is only ever one key in this dict
```

Comment:
This assumption doesn't hold for production data. There are 12 registrations that have more than 1 key in `registered_meta`. See https://osf.io/yacp6/ , for example.